addons/test_mail/tests/test_mail_channel.py

```diff
@@ -162,6 +162,11 @@ def test_channel_mailing_list_recipients(self):
         self.env['ir.config_parameter'].set_param('mail.catchall.domain', 'schlouby.fr')
         self.test_channel.write({'email_send': True})
         self.user_employee.write({'notification_type': 'email'})
+        # login user_employee
```

Comment:
I would put a brief explanation on why we need a `res.users.log` created by `user_employee` here (i.e. it marks it as active and logged in at least once, which is a requirement for receiving notifications).

Reply:
Changes are done. Thanks.